docs/wiki/Fetching latest updates.md

```diff
@@ -0,0 +1,53 @@
+CARML library is continuously improved and updated.
+
+This section provides an overview on how to get your library updated.
+
+## CARML updates' impact on your copy of the library
+
+Updates can have different impacts on your version of the library:
+- impacts limited to the CI environment
+
+    a general **pipeline update** can be done by copying/replacing any code outside of the `modules` folder. Refer to [Upgrade from 0.3.1 to 0.4.0](./Upgrade%20from%200.3.1%20to%200.4.0#general-pipeline-updates) as an example.
+
+- impacts limited to specific modules
+
+    new created modules or changed ones can be updated by copying/replacing the content of the new `modules` folder on your own `modules` folder. Refer to [Upgrade from 0.3.1 to 0.4.0](./Upgrade%20from%200.3.1%20to%200.4.0#general-import-of-new-modules) as an example.
```

Comment:
I don't think we should refer to the specific upgrade from 0.3.1. to 0.4.0 in this general section, it should, in case, be the other way around.

Reply:
removed specific upgrade considerations